clinica/pipelines/t1_freesurfer_atlas/t1_freesurfer_atlas_utils.py

```diff
@@ -21,45 +101,91 @@ def compute_atlas(
     for path in Path(path_to_atlas).rglob(
         "*" + to_process_with_atlases[0] + "_6p0.gcs"
     ):
-        hemisphere = Path(path.stem).stem
-        atlas_name = Path(path.stem).suffix[1:].split("_")[0]
-        sub, ses = to_process_with_atlases[1].split("_")
-        subject_dir = (
-            caps_directory
-            + "/subjects/"
-            + sub
-            + "/"
-            + ses
-            + "/t1/freesurfer_cross_sectional"
-        )
+        if len(to_process_with_atlases[1].split("_")) == 2:
```

Comment:
can you add a comment to indicate what these checks (i.e `==2 or ==3`) correspond to?

Reply:
Done !

Reply:
Maybe do a check on "long" instead, as it seems clearer (i.e `if "long" in string:`)